test/fixtures/reference_books.yml

```
@@ -0,0 +1,9 @@
+reference_book1:
```

```suggestion
reference_book_1:
```

他のfixtureはこうなっているので一貫性を維持したいです。